starter-code/css/main.css

```diff
@@ -3,6 +3,149 @@ Colors:
 
 Text: 1A1A1A
 Green: #00B172
-White: #FFF
+White: #FFF*/
+
+nav{
+    position: fixed;
+    display: flex;
+    justify-content: space-between;    
+    width: 96%;
+    padding: 10px 2%;
+    background-color: white;
+    align-items: center;
+    font-size: 25px;
+  }
+   nav img{
+    width: 19%;
+    height:243px;
+    float: left;
+   }
+ 
+#menu{
+    display: flex;
+  }
+
+
+#menu  li{
+    display: flex;
+    justify-content: space-around;
+    font-size: 70px;
+    list-style: none;
+    margin: 60px;
+    padding-top: 16px
+ 
+}
+header{
+    height: 2100px;
+}
+#container{
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 850px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
+}
+#centered{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform:translate(-50%,-50%);
+}
+h1{
+    font-size: 238px;
+    text-align: center;
+}
+h3{
+    
+    font-size: 75px;
+    text-align: center;
+}
+h4{
+    
+    font-size: 75px;
+    text-align: center;
+}
+#sectionone{
```

Comment:
be careful with using ID's. ID's shouldn't be used for styling and can only be used once